app-cdr/iat/iat-0.1.7-r2.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2016 Gentoo Foundation
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 DESCRIPTION="BIN, MDF, PDI, CDI, NRG, and B5I converters"
-HOMEPAGE="https://wiki.gentoo.org/wiki/No_homepage"
-SRC_URI="mirror://gentoo/${P}.tar.bz2"
-
+HOMEPAGE="https://www.berlios.de/software/iso9660-analyzer-tool"
+SRC_URI="mirror://sourceforge.net/projects/iat.berlios/files/iat-${PV}.tar.bz2"
```

Comment:
This should be SRC_URI="https://download.sourceforge.net/iat.berlios/${P}.tar.bz2"